mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -194,6 +194,11 @@ def get_model_version_dependencies(model_dir):
         )
         for endpoint_name in endpoint_names:
             dependencies.append({"type": "DATABRICKS_MODEL_ENDPOINT", **endpoint_name})
+        function_names = _fetch_langchain_dependency_from_model_info(
```

Comment:
Seems we can create a helper function for these lines as it's repeated three times for different resource types. BTW if we could make "DATABRICKS_UC_FUNCTION" constant then it's easier to maintain. Not a blocker though.